test/index.test.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+import assert from 'assert';
+import Render from '../app/render';
+import { describe, it, before } from 'mocha';
+
+describe('Vue unit test', () => {
+  before(() => {
+    document.body.innerHTML = window.__html__['fixtures/test.html'];
```

Comment:
Sry, this project doesn't use `karma-fixture` [package.json](https://github.com/uncovertruth/userdive-tag-paster/pull/86/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R40)